pmm-app/src/pmm-qan/panel/components/Details/Metadata/Metadata.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Overlay } from 'shared/components/Elements/Overlay/Overlay';
+import { Collapse, useStyles2 } from '@grafana/ui';
+import { Table } from 'shared/components/Elements/Table';
+import { LinkTooltip } from 'shared/components/Elements/LinkTooltip/LinkTooltip';
+import { getStyles } from './Metadata.styles';
+import { metadataTableHead } from './Metadata.constants';
+import { OVERLAY_LOADER_SIZE } from '../Details.constants';
+
+const Metadata = ({ metadata, loading }) => {
+  const styles = useStyles2(getStyles);
+  const [metadataOpen, setMetadataOpen] = useState(true);
+
+  const nameColumn = (item) => (
+    <span className={styles.metadataColumn}>
+      <span>{item.name}</span>
+    </span>
+  );
+
+  const valueColumn = (item) => (
+    <span className={styles.metadataColumn}>
+      <span>{item.value}</span>
+      <LinkTooltip tooltipText={item.tooltip} icon="info-circle" />
+    </span>
+  );
+
+  const columns = [
+    {
+      Header: metadataTableHead.first,
+      accessor: nameColumn,
+    },
+    {
+      Header: metadataTableHead.second,
+      accessor: valueColumn,
+    },
+  ];
+
+  return (
+    <Overlay size={OVERLAY_LOADER_SIZE}>
+      <Collapse
+        collapsible
+        label={metadataTableHead.main}
+        isOpen={metadataOpen}
+        onToggle={() => setMetadataOpen(!metadataOpen)}
+      >
+        <Table
+          columns={columns}
+          data={metadata}
+          loading={loading}
+          noData={null}
```

Comment:
I believe this is an optional prop, so there's no need to send null

Reply:
Removed an optional.